runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -219,6 +230,99 @@ public void onWidgetLoaded(WidgetLoaded event)
 		}
 	}
 
+	@Subscribe
+	public void itemContainerChanged(ItemContainerChanged event)
+	{
+		final ItemContainer container = event.getItemContainer();
+		observeForWintertodtLootEntry(container);
+	}
+
+	private void observeForWintertodtLootEntry(ItemContainer container)
+	{
+		if (container != client.getItemContainer(InventoryID.INVENTORY))
+		{
+			return;
+		}
+
+		final Item[] items = container.getItems();
+
+		// EXTRA_SUPPLY_CRATE is the reward when you trade in items to Ignisia
```

Comment:
Either remove this comment, or put it on the method, rather than the method call